src/Presentation/Nop.Web/Views/Checkout/Confirm.cshtml

```diff
@@ -73,6 +73,14 @@
                     <div class="buttons">
                         <input type="submit" name="nextstep" value="@T("Checkout.ConfirmButton")" class="button-1 confirm-order-next-step-button" />
                     </div>
+					<script>
+                        $(document).on('submit', 'form', function() {
+                            var button = $(this).find('input[type="submit"]');
+                            setTimeout(function() {

```

Comment:
sure. you're right. It should be good. For multi-step checkout.

But it misses one-page checkout page. This is what I meant in the previous comment (one-page checkout)

I would also use a certain button ID selector (some validation). This way not other submit buttons are disabled on the page.